requirements.txt

```diff
@@ -1,8 +0,0 @@
-Flask==0.12
-flake8==3.2.1
-pytest==3.0.6
-requests==2.18.3
-czech-sort==0.4
-PyYAML==3.12
-python-slugify==1.2.1
-responses==0.7.0
```

Comment:
This file is still referenced in the documentation (README). Please update the docs according to change in this commit (and preferably squash the change to this commit).